api/v1alpha1/types.go

```diff
@@ -16,11 +16,44 @@ limitations under the License.
 
 package v1alpha1
 
+import (
+	platformv1alpha1 "github.com/openshift/api/platform/v1alpha1"
+)
+
 var (
 	TypeApplied = "Applied"
 
 	ReasonSourceFailed    = "SourceFailed"
+	ReasonUnpackPending   = "ApplyPending"
 	ReasonApplyFailed     = "ApplyFailed"
 	ReasonApplySuccessful = "ApplySuccessful"
 	ReasonApplyPending    = "ApplyPending"
 )
+
+const (
+	SourcedBundleAnnotation = "platform.openshift.io/sourced-bundle"
+)
+
+func GetDesiredBundle(po *platformv1alpha1.PlatformOperator) string {
+	annotations := po.GetAnnotations()
+	desiredBundle, ok := annotations[SourcedBundleAnnotation]
+	if ok {
+		return desiredBundle
+	}
+	return ""
+}
+
+func SetDesiredBundle(po *platformv1alpha1.PlatformOperator, desiredBundle string) {
+	annotations := po.GetAnnotations()
+	if annotations == nil {
+		annotations = make(map[string]string)
+	}
+	annotations[SourcedBundleAnnotation] = desiredBundle
+	po.SetAnnotations(annotations)
+}
+
+func SetActiveBundleDeployment(po *platformv1alpha1.PlatformOperator, name string) {
+	po.Status.ActiveBundleDeployment = platformv1alpha1.ActiveBundleDeployment{
```

Comment:
What if the PO sent to this is nil?

Reply:
Updated the implementation in #41 to panic if the PO is nil.